runner.yml.dist

```diff
@@ -0,0 +1,43 @@
+drupal:
+  root: "build"
+  base_url: "http://web:8080/build"
+  site:
+    name: "OpenEuropa"
+    profile: "minimal"
+  database:
+    host: "mysql"
+    port: "3306"
+    name: "oe_contact_forms"
+    user: "root"
+    password: ""
```

Comment:
To prevent install from failing please see sparql config here https://github.com/openeuropa/rdf_skos/blob/master/runner.yml.dist